MediaBrowser.Model/Notifications/NotificationOption.cs

```diff
@@ -15,6 +15,14 @@ public NotificationOption(string type)
             SendToUsers = Array.Empty<string>();
         }
 
+        public NotificationOption()
+        {
+            Type = string.Empty;
+            DisabledServices = Array.Empty<string>();
+            DisabledMonitorUsers = Array.Empty<string>();
+            SendToUsers = Array.Empty<string>();
+        }
+
         public string Type { get; set; }
```

Comment:
```suggestion
        public string? Type { get; set; }
```

Reply:
Didn't use a nullable as I don't know if the functions that use this support a null value for type.
Also, since Type would never be null if the original constructor was used, I didn't want to introduce any future bugs.

Although the new constructor would be used for serialization at present, you never know what someone else will do in the future - which could introduce a null value into the code.

Reply:
In my testing the serializer can still set the value of a non-nullable string to null. Until we have proper Dtos, we will have to set the type of Type to be nullable.

```c#
using System;
using System.Text.Json;

namespace Test
{
    public static class Program
    {
        private static void Main()
        {
            const string nullable = @"{ ""Str"": null }";
            const string nullable2 = @"{}";
            const string set = @"{ ""Str"": """" }";

            var object1Null = JsonSerializer.Deserialize<Obj1>(nullable);
            Console.WriteLine(JsonSerializer.Serialize(object1Null));

            var object1Blank = JsonSerializer.Deserialize<Obj1>(nullable2);
            Console.WriteLine(JsonSerializer.Serialize(object1Blank));
            
            var object1String = JsonSerializer.Deserialize<Obj1>(set);
            Console.WriteLine(JsonSerializer.Serialize(object1String));

            var object2Null = JsonSerializer.Deserialize<Obj2>(nullable);
            Console.WriteLine(JsonSerializer.Serialize(object2Null));
            
            var object2Blank = JsonSerializer.Deserialize<Obj2>(nullable2);
            Console.WriteLine(JsonSerializer.Serialize(object2Blank));
            
            var object2String = JsonSerializer.Deserialize<Obj2>(set);
            Console.WriteLine(JsonSerializer.Serialize(object2String));
        }
    }

    public class Obj1
    {
        public string Str { get; set; }

        public Obj1()
        {
            Str = string.Empty;
        }
    }

    public class Obj2
    {
        public string? Str { get; set; }

        public Obj2()
        {
            Str = null;
        }
    }
}
```

output:
```
{"Str":null}
{"Str":""}
{"Str":""}
{"Str":null}
{"Str":null}
{"Str":""}
```